### PR TITLE
Add barebones account system

### PR DESCRIPTION
**Pull request by azertyfun**
Please read [the backend PR first](https://github.com/pebble-dev/rebblestore-api/pull/28).

This is a frontend implementation for the account system. I have no previous Vue.js experience, so some of the code design choices are probably questionable, and the UI doesn't quite integrate like it should with the current UI, so that will have to be fixed at some point.

### CODE REVIEW

**Comment by Zetaphor** [comment]
@sGerli Will review this tonight

File: src/App.vue [hunk 1]
```diff
@@ -25,8 +25,131 @@ export default {
...
+      authProvider: window.localStorage.getItem('authProvider'),
+      accountInformation: {
+        loggedIn: false,
+        name: 'Guest'
+      },
+      authProviders: {
+        google: {
+          name: 'google',
+          scopes: 'profile email',
+          client_id: '', // Insert client ID here
```
Comment (sGerli) [inline src/App.vue]: Can this be fetched from backend?

File: src/components/pages/AccountLogin.vue [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        <template v-else-if="errors.n === 0">
+          Logging in...
+        </template>
+      </div>
+    </main>
+  </div>
+</template>
+
+<script>
+import VueRecaptcha from 'vue-recaptcha'
```
Comment (sGerli) [inline src/components/pages/AccountLogin.vue]: You forgot to remove this import

**Review by sGerli — CHANGES_REQUESTED** [review]

File: src/components/pages/AccountLogin.vue (hunk 2 above)
Comment (azertyfun) [reply src/components/pages/AccountLogin.vue]: Well spotted, removing it

File: src/App.vue (hunk 1 above)
Comment (azertyfun) [reply src/App.vue]: Actually yes, on it right now

**Comment by sGerli** [comment]
No longer relevant

**Pull request closed without merging**